plugins/go-build/share/go-build/1.14.5

```diff
@@ -0,0 +1,11 @@
+install_darwin_64bit "Go Darwin 64bit 1.14.5" "https://golang.org/dl/go1.14.5.darwin-amd64.tar.gz#3fa7ed8dc44fdd50c0bfe72676250cceca527d59950aef20af906a670cf88de2"
```

Comment:
they changed the hash to `1d01212ac86340e6680cb36e354339920e7c6e99b4e09a9d1377a2cd2ca5f760`

https://gomirrors.org/

Reply:
Thanks karuna! Updated.